packages/grpc-native-core/ext/channel_credentials.cc

```diff
@@ -58,6 +60,43 @@ ChannelCredentials::~ChannelCredentials() {
   grpc_channel_credentials_release(wrapped_credentials);
 }
 
+struct callback_md {
+    Nan::Callback *callback;
+};
```

Comment:
Why did you use a struct here, instead of using the `callback` pointer directly?